flink-runtime/src/main/java/org/apache/flink/runtime/security/SecurityUtils.java

```diff
@@ -230,10 +231,21 @@ private void validate() {
 		}
 
 		private static List<String> parseList(String value) {
-			if(value == null) {
+			if(value == null || value.isEmpty()) {
 				return Collections.emptyList();
 			}
-			return Arrays.asList(value.split(","));
```

Comment:
I think can we using regular expression: `return Arrays.asList(value.replaceAll("\\s*,\\s*", ",").split(","));`